TF_Service_dotNet/TouchFree/Interactions/PositionStabiliser.cs

```diff
@@ -20,16 +20,31 @@ public class PositionStabiliser : IPositionStabiliser
         private Vector2 previousPositionDeadzoneDefaultSize;
         private Vector2 previousPositionDeadzoneCurrentSize;
 
+        private Vector2 deadzoneOffset;
+        Vector2 lastRawPos;
+
+        public void SetDeadzoneOffset()
+        {
+            deadzoneOffset = previousPositionDeadzoneCurrentSize - lastRawPos;
+        }
+
+        public void ReduceDeadzoneOffset()
+        {
+            deadzoneOffset *= 0.9f;
+        }
```

Comment:
Can we discuss whether adding the deadzone offset should exist as a separate PR or if it is okay to be included here? @PeterNancollis-UH @NatGreenwood-UL

Reply:
I can back this change out no problem. It's not fully implemented yet either as none of the interactions have been set to use it.

Reply:
If it's not used currently then yeah, I think it best to keep it for a later PR. Happy to approve after that :)